matplotlib-tutorial.ipynb

```diff
@@ -89,6 +89,7 @@
    },
```

Comment:
<p>This is an unneccessary comment. We all know Python.</p>

 _Reply via <a href='https://app.reviewnb.com/sebinsua/test-review-notebooks/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Am I replying to myself here?</p>